azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/MessageSender.java

```diff
@@ -828,7 +828,7 @@ private void throwSenderTimeout(final CompletableFuture<Void> pendingSendWork, f
 
         final boolean isClientSideTimeout = (cause == null || !(cause instanceof EventHubException));
         final EventHubException exception = isClientSideTimeout
-                ? new TimeoutException(String.format(Locale.US, "%s %s %s.", MessageSender.SEND_TIMED_OUT, " at ", ZonedDateTime.now(), cause))
```

Comment:
why - remove the cuase - exception?

Reply:
Yes. i will add cause with one more formatter %s to String.format. Earlier no of argument to String.format and '%s' were not matching.